Pipfile

```diff
@@ -12,6 +12,10 @@ whitenoise = "6.2.0"
 gunicorn = "20.1.0"
 djangorestframework = "3.13.1"
 drf-yasg = "1.20.0"
+typing-extensions = "4.2.0"
+djangorestframework-simplejwt = "*"
+mypy = "*"
+django-rest-passwordreset = "*"
```

Comment:
This package is not actively maintained, avoid using third party packages where possible, it might pose security risks or incompatibility issues in the future, you can ask  Dalienst how he override the Django PasswordTokenGenerator class

Reply:
Okay